blogs/serializers.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth.models import User
+from rest_framework import serializers
+
+from authentication.serializers import *
```

Comment:
Why are we using wildcard imports?
Please fix this

Reply:
idk what that does so i removed it.